CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
```

Comment:
duly noted.

laid down another commit.  i'll wait to rebase again until all the QA gods are appeased :smile: